src/sentry/tasks/store.py

```diff
@@ -409,6 +409,11 @@ def _do_process_event(
     with sentry_sdk.start_span(op="tasks.store.process_event.get_project_from_cache"):
         project = Project.objects.get_from_cache(id=project_id)
 
```

Comment:
Can we skip this if it's already set something like `if getattr(project,  '_organization_cache')`? Or that can't be the case here with project coming from cache?

Reply:
We cannot trust the cache value if there is one. the cache invalidation is not taking associated models into account

Reply:
I see. Thanks for explanation.